spec/jobs/reports/monthly_account_reuse_report_spec.rb

```diff
@@ -25,17 +27,24 @@
   end
 
   describe '#perform' do
+    let(:body_of_report) do
+      "\"{:title=>\"\"IDV app reuse rate Feb-2021\"\", :float_as_percent=>true, :precision=>4}\","\
+      "\"[\"\"Num. SPs\"\", \"\"Num. users\"\", \"\"Percentage\"\"]\",\"[\"\"Total (all >1)\"\", "\
+      "0, 0]\"\n\"{:title=>\"\"Total proofed identities\"\"}\",\"[[\"\"Total proofed identities "\
+      "(Feb-2021)\"\"], [0]]\"\n"
+    end
```

Comment:
1. I don't think we should be testing the exact content of the stuff that gets written to S3
2. the stuff for the mailer (the title row, etc etc) probably does not belong in S3

I feel like as a consumer of the report I would expect each table to be in a separate `.csv` file in S3 in a subfolder by day?

Reply:
I’ve cleaned up the spec file, but I’m moving the S3 organization to a new ticket: https://cm-jira.usa.gov/browse/LG-11116